### PR TITLE
fix(cli): don't traverse fs if started in a random folder

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: src/Playwright.CLI/Program.cs [hunk 1]
```diff
@@ -32,21 +32,41 @@ static class Program
...
+                    path = Path.Combine(path, "..");
+                }
+
                 args = args[2..];
             }
 
-            var file = Traverse(new DirectoryInfo(root));
+            if (File.GetAttributes(path).HasFlag(FileAttributes.Directory))
+            {
+                var solutions = Directory.GetFiles(path, "*.sln");
```
Comment (avodovnik) [inline src/Playwright.CLI/Program.cs:55]: I'm not sure about this... It's possible you'd clone a repo with a `src` folder, and you'd be in the root folder.

File: src/Playwright.CLI/Program.cs [hunk 2]
```diff
@@ -32,21 +32,41 @@ static class Program
...
+                var projects = Directory.GetFiles(path, "*.*proj");
+                if (solutions.Length == 0 && projects.Length == 0)
+                {
+                    return PrintError($"Couldn't find project using Playwright. Ensure a project or a solution exists in {path}, or provide another path using -p.");
```
Comment (avodovnik) [inline src/Playwright.CLI/Program.cs:59]: Nit: it's the same string as above, maybe unify the check/error somewhere?

File: src/Playwright/Program.cs [hunk 3]
```diff
@@ -47,8 +47,7 @@ public int Run(string[] args)
             }
             catch (Exception)
             {
-                Console.WriteLine("Microsoft.Playwright is missing required assets. Please ensure to build your project before running.");
-                return 1;
+                return PrintError("Microsoft.Playwright build was found, but is missing required assets. Please ensure to build your project before running Playwright tool.");
```
Comment (avodovnik) [inline src/Playwright/Program.cs:50]: Not sure what `build was found` means in this context.  Maybe package?
```suggestion
                return PrintError("Microsoft.Playwright package was found, but is missing required assets. Ensure your project is built, before running Playwright tool.");
```

File: src/Playwright.CLI/Program.cs (hunk 1 above)
Comment (pavelfeldman) [reply src/Playwright.CLI/Program.cs:55]: None of the dotnet build / test / run would work in that case, so I guess it is expected that you pass -p...

File: src/Playwright.CLI/Program.cs (hunk 2 above)
Comment (pavelfeldman) [reply src/Playwright.CLI/Program.cs:59]: M: Maturity

File: src/Playwright/Program.cs (hunk 3 above)
Comment (pavelfeldman) [reply src/Playwright/Program.cs:50]: assembly?

**Pull request merged**